src/PensionsRegulator.Functions/Extensions/FunctionsHostBuilderConfigurationsExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace PensionsRegulator.Functions
+{
+    internal static class FunctionsHostBuilderConfigurationsExtensions
```

Comment:
Has this class been copied from elsewhere?  It looks like something we're probably using in multiple places so could be shared?